megamek/src/megamek/client/ui/swing/CommonSettingsDialog.java

```diff
@@ -320,34 +339,13 @@ public CommonSettingsDialog(JFrame owner, ClientGUI cg) {
      * @param owner - the <code>Frame</code> that owns this dialog.
      */
     public CommonSettingsDialog(JFrame owner) {
-        // Initialize our superclass with a title.
-        super(owner, Messages.getString("CommonSettingsDialog.title")); //$NON-NLS-1$
-
-        panTabs = new JTabbedPane();
-
-        JPanel settingsPanel = getSettingsPanel();
-        JScrollPane settingsPane = new JScrollPane(settingsPanel);
-        panTabs.add("Main", settingsPane);
-
-        JPanel tacticalOverlaySettingsPanel = getTacticalOverlaySettingsPanel();
-        JScrollPane tacticalOverlaySettingsPane = new JScrollPane(tacticalOverlaySettingsPanel);
-        panTabs.add("Graphics", tacticalOverlaySettingsPane);
-
-        JPanel keyBindPanel = getKeyBindPanel();
-        JScrollPane keyBindScrollPane = new JScrollPane(keyBindPanel);
-        panTabs.add("Key Binds", keyBindScrollPane);
-
-        JPanel buttonOrderPanel = getButtonOrderPanel();
-        panTabs.add("Button Order", buttonOrderPanel);
-        
-        JPanel advancedSettingsPanel = getAdvancedSettingsPanel();
-        JScrollPane advancedSettingsPane = new JScrollPane(advancedSettingsPanel);
-        panTabs.add("Advanced", advancedSettingsPane);
+        super(owner, Messages.getString("CommonSettingsDialog.title"), true);
 
+        JTabbedPane panTabs = new JTabbedPane();
         setLayout(new BorderLayout());
         getContentPane().add(panTabs, BorderLayout.CENTER);
         getContentPane().add(getButtonsPanel(), BorderLayout.PAGE_END);
-
+        
```

Comment:
```suggestion

```

Reply:
Not sure what's wrong with this line. No tabs. A bit of breathing space for the code ;)

Reply:
This removes the extra spacing from it, which would be automatically removed in a further commit. Just dealing with future blankspace changes now.